pkg/diskboot/config.go

```diff
@@ -63,7 +64,7 @@ type Entry struct {
 
 // KexecLoad calls the appropriate kexec load routines based on the
 // type of Entry
-func (e *Entry) KexecLoad(mountPath, appendCmdline string, dryrun bool) error {
+func (e *Entry) KexecLoad(mountPath, appendCmdline string, removeCmdline, reuseCmdline []string, dryrun bool) error {
```

Comment:
I don't think these should be arguments to KexecLoad. I think the user of this package should do this before even getting here. (I say this because I don't think it should be an argument to boot.OSImage.Load.)

Reply:
yes rigth on pxeboot I was doing: ```
img.Cmdline = updateBootCmdline(img.Cmdline)
```
But this KexecLoad already had a `appendCmdline` parameter... anyway with the later change on using the `boot` package this will change again

Reply:
refactored to pass a single `filter` parameter